### PR TITLE
tachidesk-server: Add version 0.4.5

### PR DESCRIPTION
**Pull request by Witchilich**

### CODE REVIEW

**Comment by Witchilich** [comment]
Seems to work fine locally in my PC.

File: bucket/tachidesk.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "description": "A rewrite of Tachiyomi for the Desktop",
```
Comment (Ash258) [inline bucket/tachidesk.json:2]: Completely nothing-saying and nonsense description. For end user absolutely unusable description

File: bucket/tachidesk.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "homepage": "https://github.com/Suwayomi/Tachidesk",
+    "license": {
+        "identifier": "Mozilla Public License 2.0",
```
Comment (Ash258) [inline bucket/tachidesk.json:5]: Invalid SPDX license identifier.

File: bucket/tachidesk.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        "url": "https://github.com/Suwayomi/Tachidesk/blob/master/LICENSE"
+    },
+    "url": "https://github.com/Suwayomi/Tachidesk/releases/download/v0.2.7/Tachidesk-v0.2.7-r312-win32.zip",
+    "hash": "3b97e60ffef0555a101a86cb3c5a6f7b86001d8dccc4861f77522b62c5ff8984",
+    "extract_dir": "Tachidesk-v0.2.7-r312-win32",
+    "version": "0.2.7",
```
Comment (Ash258) [inline bucket/tachidesk.json:11]: Put version as a first property

File: bucket/tachidesk.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            "Tachidesk"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Suwayomi/Tachidesk"
+    },
+    "autoupdate": {
+        "url": "https://github.com/Suwayomi/Tachidesk/releases/download/v$version/Tachidesk-v$version-$matchCommit-win32.zip",
+        "extract_dir": "Tachidesk-v$version-$matchCommit-win32"
+    },
```
Comment (Ash258) [inline bucket/tachidesk.json:30]:
```suggestion
    }
```

File: bucket/tachidesk.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            "server.exe",
+            "tachidesk"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "server.exe",
+            "Tachidesk"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/tachidesk.json:23]: Does not make sense to have shortcut for it as this is clearly meant to be executed as a service

```suggestion
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/tachidesk.json (hunk 5 above)
Comment (Witchilich) [reply bucket/tachidesk.json:23]: You are supposed to execute server.exe. That is the executable for Tachidesk which then opens Tachidesk on your browser. Its based of TachiWeb so they still call it server.exe.

File: bucket/tachidesk.json (hunk 3 above)
Comment (Ash258) [reply bucket/tachidesk.json]: Put version as a **first** property

File: bucket/tachidesk.json (hunk 1 above)
Comment (Witchilich) [reply bucket/tachidesk.json]: That was based off their repo about section. So here goes the new one.

**Comment by Witchilich** [comment]
Everything about this app(including the location) has changed since the original pull request. Have to change those script locations from patch-1 to extras.

**Comment by issaclin32** [comment]
I updated the manifest by myself, because the PR has been hanging for a while.

**Review by rashil2000 — APPROVED** [review]

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**